test/integration/ssh_server/arch_opkssh.Dockerfile

```
@@ -0,0 +1,80 @@
+# Stage 1: Build the Go binary
+FROM golang:1.23 as builder
+
+# Set destination for COPY
+WORKDIR /app
+
+# Download Go modules
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy our repo
+COPY . ./
+
+# Copy the source code and build the binary
+ARG ISSUER_PORT="9998"
+RUN go build -v -o opksshbuild
+
+# Stage 2: Create a minimal ArchLinux-based image
+FROM quay.io/archlinux/archlinux
+# Install dependencies required for runtime (e.g., SSH server)
+RUN pacman -Syu && \
+    pacman -Sy openssh inetutils wget jq --noconfirm && \
```

```suggestion
    pacman -Sy openssh inetutils wget jq sudo --noconfirm && \
```

It looks like the integration test is breaking because sudo isn't installed. This might fix it